app/views/students/_form.html.erb

```diff
@@ -1,16 +1,25 @@
 <%= form_for @student do |f| %>
+
+  <div class= "error-message">
+    <% if @student.errors.any? %>
+        <% @student.errors.full_messages.each do |msg| %>
+          <%= msg %>
+        <% end %>
+    <% end %>
+  </div><br>
+
   <div>
-    <%= f.label :name %><br>
-    <%= f.text_field :name %><br>
-  </div>
+    <%= f.label :name, prompt: true %><br>
+    <%= f.text_field :name %>
+  </div><br>
```

Comment:
you could add a class. e.g.
`<div class="student-form-item">
and then add a breaking space afterwards...

but what you have is fine too.